drupal/rootfs/etc/islandora/utilities.sh

```diff
@@ -336,9 +369,7 @@ function update_settings_php {
     fi
 
     # Restore owner/group to previous value
-    if [ ! -z "${previous_owner_group}" ]; then
-        chown "${previous_owner_group}" "${site_directory}/settings.php"
-    fi
+    restore_settings_ownership ${site} ${previous_owner_group}
 
     # Restrict access to settings.php
     chmod 444 "${site_directory}/settings.php"
```

Comment:
We should move the correction of permissions into `restore_settings_ownership`
```bash
# Restrict access to settings.php
chmod 444 "${site_directory}/settings.php"
```

Reply:
Yes - good catch. I've made this change.